lib/utils/objects.ts

```diff
@@ -1,19 +1,53 @@
+import * as t from "io-ts";
+
+export const isObject = (o: {}) =>
```

Comment:
btw `utils/types` is meant to be a container for all type-related function (e.g. type checking, io-ts helpers, etc...) pls consider moving everything in there to avoid confusion